overlay/buildroot/index.html

```diff
@@ -0,0 +1,221 @@
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="x-ua-compatible" content="ie=edge">
+<title>HNAP4PlutoSDR On-board documentation</title>
+<meta name="description" content="HNAP4PlutoSDR On board docs">
+
+<link rel="apple-touch-icon" href="apple-touch-icon.png">
+<link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+<!-- Place favicon.ico in the root directory -->
+<link type="text/css" href="./img/style.css" rel="stylesheet">
+</head>
+<body>
+<!--[if lte IE 11]>
+<p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+<![endif]-->
+
+<!-- Add your site or application content here -->
+<header id="top">
+<a href="https://hnap.de">
+<img src="img/hnap_logo.png" alt="HNAP logo" />
+</a>
+<div class="anchor">
+<h1>HNAP4PlutoSDR</h1>
+</div>
+</header>
+
+<nav style="text-align: center;">
+<div class="dropdown">
+<span><a class="button" href="#firmware">Firmware &dArr;</a></span>
+<div class="dropdown-content">
+<p><a class="button" href="#firmware">Upgrade Firmware</a></p>
+<p><a class="button" href="#version">Version Info</a></p>
+<p><a class="button" href="#target">Custom Apps</a></p>
+<p><a class="button" href="#config">Config Settings</a></p>
+<p><a class="button" href="#wifi">Wi-Fi Settings</a></p>
+<p><a class="button" href="#wired">Wired Settings</a></p>
+</div>
+</div>
+<a class="button" href="#support">Help and Support</a>
+<a class="button" href="LICENSE.html">License</a>
+</nav>
+<hr>
+<section id="firmware">
+<a class="anchor" href="#top">Back to top</a>
+<h3>Firmware</h3>
+<div id="versionsection" class="download">
+<p><strong>Status of the HNAP4PlutoSDR firmware:</strong></p>
+<p><span id="versiontest">Need javascript to check (sorry)</span></p>
```

Comment:
The script that replaces this message encounters an error (see below). My Browser only displays the "Need Javascript to check" message.